coursemapper-kg/recommendation/app/services/course_materials/recommendation/wikipedia_service.py

```
@@ -34,7 +35,8 @@
                 # print("w_data", data)
 
             except (PageError, DisambiguationError) as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)